internal/rest/client/helper.go

```diff
@@ -0,0 +1,97 @@
+/*
+	Copyright NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package client
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/edge-api/rest_client_api_client"
+	"github.com/openziti/edge-api/rest_client_api_client/current_api_session"
+	"github.com/openziti/edge-api/rest_client_api_client/external_jwt_signer"
+	"github.com/openziti/edge-api/rest_model"
+	"github.com/openziti/edge-api/rest_util"
+	internalconsts "github.com/openziti/ziti/internal/rest/consts"
+	"github.com/openziti/ziti/ziti/util"
+	"net/http"
+	"os"
+)
+
+func NewClientApiClient() (*rest_client_api_client.ZitiEdgeClient, error) {
```

Comment:
I would be surprised if something like this already didn't exist in the CLI. I don't know for sure, but I sense this block keeps repeating.

Reply:
Usually all the ziti cli stuff is always using the management API, this is the first time i've personally seen/used the client api with the ziti CLI on purpose.